src/current/cockroachcloud/export-metrics.md

```diff
@@ -338,6 +338,67 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
+
+Perform the following steps to enable metrics export from your CockroachDB {{ site.data.products.dedicated }} cluster to Azure Monitor.
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID that will be used in Step 5:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. The ID should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
+
+1. Create a private key and certificate pair, using a tool such as OpenSSL. Refer to [this example]({% link {{site.current_cloud_version}}/create-security-certificates-openssl.md %}#step-1-create-the-ca-key-and-certificate-pair) to create a private key file (`ca.key`) and a certificate file (`ca.crt`) that will be uploaded in Step 3.
+
+  1. Combine the certificate and key files using the following command. The string in the combined file will be used in Step 5.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    cat ca.crt ca.key | awk '{printf "%s\\n", $0}'  > combined.pem
+    ~~~
+
+1. [Create an Entra application in Azure](https://learn.microsoft.com/en-us/entra/identity-platform/howto-create-service-principal-portal). When registering the application, leave the optional **Redirect URI** empty since certificate-based authentication is being used, not browser-based authentication.
+
+  1. Once the application is registered, upload the certificate file (`ca.crt`) created in Step 2.
+  1. From the **Overview** page for the Entra application, copy the values for *Display Name*, *Application (client) ID*, and the *Directory (tenant) ID* to be used in Step 5.
+
+1. [Create an Azure Monitor Application Insights resource](https://learn.microsoft.com/en-us/azure/azure-monitor/app/create-workspace-resource).
+
+  1. In the newly-created Application Insights resource, add a role assignment that assigns the Entra application (search for *Display Name* from Step 3) to the role [Monitoring Metrics Publisher](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/monitor#monitoring-metrics-publisher).
+  1. From the **Overview** page for the Application Insights resource, from the *Connection String*, copy the values for the *InstrumentationKey* and *IngestionEndpoint* to be used in Step 5.
```

Comment:
```suggestion
  1. From the **Overview** page for the Application Insights resource, view the *Connection String*. Copy the values for *InstrumentationKey* and *IngestionEndpoint*, which will be used in Step 5.
```
I'm wondering why these are italicized?

Reply:
updated. I changed the italics to bold because they are UI elements.